seq2seq/third_party/models/configuration_t5.py

```diff
@@ -14,8 +14,8 @@
 # limitations under the License.
```

Comment:
This comment is about adding text to line 3.

Can you add a comment as the doc I sent you discusses (indicating this file has been modified etc.)?

Reply:
thanks, done

Reply:
Great, looks good. Please add a new line back in for clarify before the License part.